girder/api/v1/system.py

```diff
@@ -295,6 +295,9 @@ def discardPartialUploads(self, uploadId, userId, parentId, assetstoreId,
         .errorResponse('You are not a system administrator.', 403)
     )
     def restartServer(self, params):
+        if not config.getConfig()['server'].get('cherrypy_server', True):
+            raise RestException('Restarting of server is disabled.')
```

Comment:
It's not really the client's fault if this operation is disallowed, so I don't think `400 Bad Request` is the appropriate error code. IMO, `403 Forbidden` is the most appropriate, since this is like:
* ([from the spec](https://tools.ietf.org/html/rfc7231#section-6.5.3)) `the server understood the request but refuses to authorize it` ... `a request might be forbidden for reasons unrelated to the credentials`
* [from the Wikipedia article](https://en.wikipedia.org/wiki/HTTP_403#Difference_from_status_.22401_Unauthorized.22) (which is admittedly less definitive) `The operation is forbidden to all users. For example, requests for a directory listing return code 403 when directory listing has been disabled.`

Of course, I don't think this is that critical, since Girder's error codes aren't always very compliant with HTTP.